examples/scripts/sft_trainer.py

```diff
@@ -81,7 +81,8 @@ class ScriptArguments:
 mlflow.log_param('num_params', num_params)
 
 # Step 2: Load the dataset
-dataset = load_dataset(script_args.dataset_name_or_path, split="train")
+train_dataset = load_dataset(script_args.dataset_name_or_path, split="train")
+eval_dataset = load_dataset(script_args.dataset_name_or_path, split="test")
```

Comment:
Is it necessary to have this eval dataset since we are only going to use train_loss for metric?